mw-webapp/src/component/text/Text.module.scss

```diff
@@ -0,0 +1,17 @@
+.Text {
```

Comment:
No need create empty styles for the future

Reply:
delete file text.module.cscc